docs/en/api/POSIX-API.md

```diff
@@ -140,6 +162,34 @@ pid mount_point alluxio_path
 
 ## Advanced Setup
 
+### Fuse in worker process
+
+Unlike standalone Fuse which you can mount at any time without Alluxio worker involves,
+the embedded Fuse has the exact same life cycle as the worker process it embeds into.
+When the worker starts, the Fuse is mounted based on worker configuration.
+When the worker ends, the embedded Fuse is unmounted automatically.
+If you want to modify your Fuse mount, change the configuration and restart the worker process.
+
+Enable the fuse in worker by setting `alluxio.worker.fuse.enabled` to `true` in the `${ALLUXIO_HOME}/conf/alluxio-site.properties`
+```
+alluxio.worker.fuse.enabled=true
+```
+
+All the information about the Fuse mount needs to be provided as the worker configuration.
+```
+alluxio.worker.fuse.mount.alluxio.path=<alluxio_path>
+alluxio.worker.fuse.mount.point=<mount_point>
+alluxio.worker.fuse.mount.options=<list of mount options separated by comma>
+```
+
+For example, the following example mounts Alluxio path `/mnt/people` to local path `/`
+with `kernel_cache,entry_timeout=7200,attr_timeout=N` mount options.
```

Comment:
is it just `N` or you are indicating a number?

Reply:
Changed to 7200, thanks!